griptape/drivers/image_generation/huggingface_diffusion_pipeline_image_generation_driver.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import io
+from abc import ABC
+from typing import Optional
+
+from attrs import define, field
+
+from griptape.artifacts import ImageArtifact
+from griptape.drivers import BaseDiffusionPipelineImageGenerationModelDriver, BaseImageGenerationDriver
+from griptape.utils import import_optional_dependency
+
+
+@define
+class HuggingFaceDiffusionPipelineImageGenerationDriver(BaseImageGenerationDriver, ABC):
```

Comment:
Wondering if we should just rename to `HuggingFacePipelineImageGenerationDriver` for similarity with the `HuggingFacePipelinePromptDriver` that uses `transformers`.